app/service-catalog/integrations/datadog.md

```diff
@@ -5,8 +5,9 @@ layout: reference
 
 products:
     - gateway
-
-description: placeholder
+works_on:
+    - konnect
+description: The Datadog integration lets you connect Datadog entities directly to your Service Catalog services.
```

Comment:
```suggestion
description: The Datadog integration lets you connect Datadog entities directly to your Service Catalog services.

breadcrumbs:
  - /service-catalog/
```
I'm not sure if this will work, but I think I'd like these to have a Service Catalog breadcrumb.